NEWS.md

```diff
@@ -5,6 +5,11 @@
 
 # Development
 
+* Remotes can be forced to use only its internal code by setting the
+  environment variable `REMOTES_NO_EXTERNAL` = "true". This is useful when
```

Comment:
I think the name is somewhat misleading, because "external" often refers in compiled code or system programs in R. Something like `R_REMOTES_STANDALONE` or  `R_REMOTES_NO_DEPS` or something like that would be better, I think.

Reply:
`R_REMOTES_STANDALONE` sounds good to me